src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -498,7 +498,7 @@ function mightCauseHorizontallyOverlappingBubbles(element) {
     //we can be very conservative and say that if the text
     //box isn't taking up the whole width, it *might* cause
     //an overlap
-    if(element.hasClass('bloom-alwaysShowBubble')) {
+    if((element.hasClass) && element.hasClass('bloom-alwaysShowBubble')) {
```

Comment:
@phillip-hopper  yes, this is fixed in my merged PR#333. I don't think this is what caused the original issue BL-1030, though, since it predates my causing the problem here. :)